modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -507,7 +508,15 @@ public String getOutputDir() {
         return outputFolder();
     }
 
-    public void setTemplateDir(String templateDir) {
+    public String getInputSpec() {
+		return inputSpec;
+	}
```

Comment:
It seems to me the code is not indented properly (using tab instead of spaces). Would you please fix it?